scripts/20170426-col482/add_activity_enums.sql

```diff
@@ -0,0 +1,13 @@
+-- Add new enum values for `type` column in `activities` table
+
+ALTER TYPE enum_activities_type ADD VALUE 'get_view_asset';
+ALTER TYPE enum_activities_type ADD VALUE 'get_whiteboard_add_asset';
+ALTER TYPE enum_activities_type ADD VALUE 'remix_whiteboard';
+ALTER TYPE enum_activities_type ADD VALUE 'get_remix_whiteboard';
+
+-- Add new enum values for `type` column in `activity_types` table
+
+ALTER TYPE enum_activity_types_type ADD VALUE 'get_view_asset';
+ALTER TYPE enum_activity_types_type ADD VALUE 'get_whiteboard_add_asset';
+ALTER TYPE enum_activity_types_type ADD VALUE 'remix_whiteboard';
+ALTER TYPE enum_activity_types_type ADD VALUE 'get_remix_whiteboard';
```

Comment:
@johncrossman It is non-trivial. The method is here if we need it:
http://stackoverflow.com/questions/25811017/how-to-delete-an-enum-type-in-postgres
but let's hope not.